docs/topics/Threads.md

```diff
@@ -0,0 +1,111 @@
+- The [Guild Create](#DOCS_TOPICS_GATEWAY/guild-create) Gateway Event will contain a [thread member](#DOCS_RESOURCES_CHANNEL/thread-member-object) object as a property on any returned threads the current is a member of.
+- The [Thread Create](#DOCS_TOPICS_GATEWAY/thread-create) Gateway Event will contain a [thread member](#DOCS_RESOURCES_CHANNEL/thread-member-object) object as a property of the thread if the current user is a member of, and the user has recently gained access to view the parent channel.
+- The [Thread List Sync](#DOCS_TOPICS_GATEWAY/thread-list-sync) Gateway Event will contain an array of [thread member](#DOCS_RESOURCES_CHANNEL/thread-member-object) objects for any returned threads the current user is a member of.
+
+### Syncing for other users
+
+> info
+> These require the `GUILD_MEMBERS` [Gateway Intent](#DOCS_TOPICS_GATEWAY/gateway-intents)
+
+- An API `GET` call to `/channels/<channel_id>/thread-members` which returns an array of [thread member](#DOCS_RESOURCES_CHANNEL/thread-member-object) objects.
+- The [Thread Members Update](#DOCS_TOPICS_GATEWAY/thread-members-update) Gateway Event which will include all users who were added to or removed from a thread by an action.
+
+## Editing & Deleting Threads
+
+Threads can be edited and deleted with the existing `PATCH` and `DELETE` endpoints to edit a channel.
+
+- Deleting a thread requires the `MANAGE_THREADS` permission.
+- Editing a thread to set `archived` to `false` only requires the current user has already been added to the thread. If `locked` is true, then the user must have `MANAGE_THREADS`
+- Editing a thread to change the `name`, `archived`, `auto_archive_duration` fields requires `MANAGE_THREADS` or that the current user is the thread creator
+- Editing a thread to change `rate_limit_per_user` or `locked` requires `MANAGE_THREADS`
+
+## NSFW Threads
+
+Threads do not explicitly set the `nsfw` field. All threads in a channel marked as `nsfw` inherit that setting though.
+
+## New Message Types
+
+Threads introduce a few new [message types](#DOCS_RESOURCES_CHANNEL/message-object-message-types), and re-purpose some others:
+
+- `RECIPIENT_ADD` and `RECIPIENT_REMOVE` have been repurposed to also send when a user is added to or removed from a thread by someone else
+- `CHANNEL_NAME_CHANGE` has been repurposed and is sent when the thread's name is changed
+- `THREAD_CREATED` is a new message sent when a thread is created from an older message, it contains a [message reference](#DOCS_RESOURCES_CHANNEL/message-reference-object-message-reference-structure) with the `guild_id` and `channel_id` of the thread. The `content` of the message is the `name` of the thread.
```

Comment:
If I understand this correctly, right now `THREAD_CREATED` is a message that appears in the parent channel that functions as a reply message to the original message and announces that a `PUBLIC_THREAD` has been created, however, [it seems possible that there may be other non-message-originating](https://github.com/discord/discord-api-docs/pull/2855/files#r622732264) thread types in the future that would need to be similarly announced into the parent channel.

These threads would not have an originating message and such the statement  `a new message sent when a thread is created from an older message` becomes not true.

Wonder if the follow edit would be more appropriate:
```suggestion
- `THREAD_CREATED` is a new message sent to the parent channel when a thread other than `PRIVATE_THREAD` is created. If the thread was created from an existing message in the parent channel, then this message will contain a [message reference](#DOCS_RESOURCES_CHANNEL/message-reference-object-message-reference-structure) with the `guild_id` and `channel_id` of the thread. The `content` of the message is the `name` of the thread.
```

Reply:
How about:
```
`THREAD_CREATED` is a new message sent to the parent `GUILD_TEXT` channel, used to inform users that a thread has been created. It is currently only sent in one case: when a `PUBLIC_THREAD` is created from an older message.
```

Reply:
could the exact requirements for "older message" be documented?

Reply:
No
But I'd be open to doing that when we launch, it's change a few times and I don't want folks depending on it
It's currently > 5 messages old or message > 10 minutes old